python/delta_sharing/reader.py

```diff
@@ -158,7 +159,11 @@ def _to_pandas(
             import delta_sharing._yarl_patch  # noqa: F401
 
         protocol = url.scheme
-        filesystem = fsspec.filesystem(protocol)
+        proxy = getproxies()
```

Comment:
are we able to test these code? it's fine to test in a real environment if it's hard to do unit test.

or could you comment on what does this do and an example of what `proxy` could look like? and why not always set `trust_env` to true?

Reply:
yes, I have tested it inhouse and also in customer env and it works as expected.

but never tested what would happen if trust_env=True is set everytime will try to test it once on what would happen.

Reply:
I have tested it again I see that even setting "trust_env" to true always also works. it would pick the HTTP proxy env variables if available else it wouldn't.

if you can also test it further

Reply:
thanks!